### DISCUSSION TITLE
NSPanel 120Pro

### DISCUSSION THREAD

**Discussion by russell2200** [NSPManager/NSPanelManager#217] [Q&A]
Can you flash the NSPanel 120P ? I understand thats it full android so a little different to the original NSPanel ?

As well the pins on the NSPanel 120P are different I can find RX, TX, GND but the other pins are elsewhere on the board and cant identify IOO

**Comment by aaronouthier** [comment]
To answer your question as stated: I'm not sure if the Pro model can be flashed or not. However, I suspect you were inquiring as to whether _this project_ can be flashed onto it. That answer is no. The two units share a similar name, but that is about where the similarities end. The hardware is completely different.

Because it runs Android, however, some have managed to "sideload" the Home Assistant app onto it, and possibly other apps. Probably not the answer you wanted, but I hope this helps. The project managers live in Norway, and are probably asleep for another few hours. They will likely know more on this subject.

**Reply by russell2200** [reply]
Awesome thanks at least confirms that it can’t be flashed with the project. Thanks

**Reply by aaronouthier** [reply]
Update: Devs are from Sweden, not Norway. That's now the 3rd time I've specified the wrong country! When will I learn?
I suppose I did think to double-check to be sure I had it right this time, so perhaps that's me learning.

Anyway. Take care!
